docs/community/com-ingest/com-ingest-flags-sql.md

```diff
@@ -56,6 +56,10 @@ molecula-consumer-sql        \
 
 ## Additional information
 
+{% include /com-ingest/com-ingest-extra-batch-size.md %}
+
+{% include /com-ingest/com-ingest-extra-concurrency.md %}
```

Comment:
Why are we adding this? Isn't this not supported? https://docs.featurebase.com/docs/community/com-ingest/com-ingest-flags-sql/